app/controllers/feedback_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class FeedbackController < ApplicationController
-  skip_before_action :access_control
+  skip_before_action :restrict_to_employee
```

Comment:
`Feedback` currently rely on `user` being present in order to submit. Minor change required in `#signed_description`.

Reply:
Interesting, my dummy_feedback seemed to work but perhaps it didn't _really_ work.

Reply:
Nope. I'm wrong. Skimmed too fast.

Reply:
Actually, sorry, what won't work about it? It seems like line 55 `return description unless user.present?` would indicate that you wrote it such that it could work for people that weren't signed in.

Reply:
I should reload before I comment :D